obsreport/obsreport_scraper.go

```diff
@@ -46,26 +65,72 @@ type ScraperSettings struct {
 }
 
 // NewScraper creates a new Scraper.
-func NewScraper(cfg ScraperSettings) (*Scraper, error) {
-	return &Scraper{
+func NewScraper(cfg ScraperSettings) *Scraper {
```

Comment:
Please don't break this API. Return error, do not log.

Reply:
Ah sorry I'm scatterbrained this week, that was a careless mistake.  Fixed this now and retested this change. Thanks for catching this!